src/app.js

```diff
@@ -85,6 +88,24 @@ app.get("/newsevents", async (request, response) => {
   renderPage(response, "newsevents");
 });
 
+//get reviews for a movie
+app.get("/api/movies/:movieId", async (request, response) => {
+  try {
+    const reviewsData = await getMovieReviews((request.params.movieId), cmsAdapterReviews);
+    const reviewArray = await paginateReviews(reviewsData, request.query.page, request.query.limit);
+
+    if (reviewArray.length > 1) {
+      response.status(200)
+      .json(reviewArray);
+    } else {
+      response.sendStatus(404)
+        .json({ message: error.message });
```

Comment:
Indentation is inconsistent on these lines. Lines 98-99 use one pattern (`.json(` aligned with `response`) and lines 101-102 use another (`.json()` indented one level compared to `response`).

I always recommend using automatic formatting to avoid these little things.